traffic_portal/app/src/common/modules/table/serverDeliveryServices/TableServerDeliveryServicesController.js

```diff
@@ -31,6 +33,32 @@ var TableServerDeliveryServicesController = function(server, deliveryServices, $
 			);
 	};
 
+	var confirmRemoveDS = function(ds, $event) {
+		if ($event) {
+			$event.stopPropagation(); // this kills the click event so it doesn't trigger anything else
+		}
+
+		var params = {
+			title: 'Remove Delivery Service from Server?',
+			message: 'Are you sure you want to remove ' + ds.xmlId + ' from this server?'
+		};
+		var modalInstance = $uibModal.open({
+			templateUrl: 'common/modules/dialog/confirm/dialog.confirm.tpl.html',
+			controller: 'DialogConfirmController',
+			size: 'md',
+			resolve: {
+				params: function () {
+					return params;
+				}
+			}
+		});
+		modalInstance.result.then(function() {
+			removeDeliveryService(ds.id);
+		}, function () {
+			// do nothing
```

Comment:
I don't like ignoring errors, can this just be a `console.error`? It might make debugging easier down the road

Reply:
same comment as above. didn't change this function. and yeah, need to circle back and fix all these "do nothing"s.